cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -348,6 +351,43 @@ describe('Unit tests for add_disaster page', () => {
   });
 });
 
+/**
+ * Set's local cookie to point to disaster with the given layers.
```

Comment:
Nit: s/Set's/Sets/. Also not actually a cookie. "Local storage"?

Reply:
Done.